src/test/javascript/spec/component/lecture-unit/lecture-unit-management.component.spec.ts

```diff
@@ -184,4 +184,22 @@ describe('LectureUnitManagementComponent', () => {
         expect(lectureUnitManagementComponent.getActionType(new VideoUnit())).toEqual(ActionType.Delete);
         expect(lectureUnitManagementComponent.getActionType(new OnlineUnit())).toEqual(ActionType.Delete);
     });
+
+    describe('isViewButtonAvailable', () => {
+        it('should return true for an attachment unit with a PDF link', () => {
+            const lectureUnit = {
+                type: LectureUnitType.ATTACHMENT,
+                attachment: { link: 'file.pdf' },
+            } as LectureUnit;
+            expect(lectureUnitManagementComponent.isViewButtonAvailable(lectureUnit)).toBeTrue();
+        });
+
+        it('should return false for an attachment unit with a non-PDF link', () => {
+            const lectureUnit = {
+                type: LectureUnitType.ATTACHMENT,
+                attachment: { link: 'file.txt' },
+            };
+            expect(lectureUnitManagementComponent.isViewButtonAvailable(lectureUnit)).toBeFalse();
+        });
```

Comment:
```suggestion
        });
        
        it('should return false for a text unit', () => {
            const lectureUnit = {
                type: LectureUnitType.TEXT,
            };
            expect(lectureUnitManagementComponent.isViewButtonAvailable(lectureUnit)).toBeFalse();
        });
```

Reply:
But people can also add files with different file extensions, no? I think it is a proper test. But when you make it LectureUnitType.TEXT, It doesn't involve uploading a file anymore

Reply:
Sorry I thought you wanted me to change the previous one, I will add this